apps/passport-server/src/services/telegramService.ts

```diff
@@ -523,25 +587,37 @@ export class TelegramService {
 
   private async sendInviteLink(
     userId: number,
-    chat: Chat.GroupGetChat | Chat.SupergroupGetChat
+    chat: Chat.GroupGetChat | Chat.SupergroupGetChat,
+    eventConfigID: string
   ): Promise<void> {
     // Send the user an invite link. When they follow the link, this will
     // trigger a "join request", which the bot will respond to.
+    const event = await fetchPretixEventInfo(
+      this.context.dbPool,
+      eventConfigID
+    );
+    if (!event) {
+      throw new Error(
+        `User used a ticket with no corresponding event confg id: ${eventConfigID}`
+      );
+    }
+
     logger(
       `[TELEGRAM] Creating chat invite link to ${chat.title}(${chat.id}) for ${userId}`
     );
     const inviteLink = await this.bot.api.createChatInviteLink(chat.id, {
       creates_join_request: true,
-      name: "test invite link"
+      name: "bot invite link"
     });
     await this.bot.api.sendMessage(
       userId,
-      `You've generated a ZK proof! Press this button to send your proof to ${chat.title}`,
+      `You've proved that you have at ticket to <b>${event.event_name}</b>!\nPress this button to send your proof to <b>${chat.title}</b>`,
       {
         reply_markup: new InlineKeyboard().url(
-          `Send ZKP`,
+          `Send Proof ✈️`,
```

Comment:
I think keeping the "ZK" is good to have, given the audience. Either "Send ZK proof" or "Send zero-knowledge proof" could work